specification/cognitiveservices/OpenAI.Inference/main.cadl

```diff
@@ -48,3 +58,10 @@ model Deployment {
   @key
   deploymentId: string;
 }
+
+@doc("Tokens input/total for a given API")
+model Usage {
```

Comment:
We should include which version this model was added to (e.g. `@Versioning.added(ServiceApiVersions.v2022-12-01)`)